package.json

```diff
@@ -1,56 +1,60 @@
 {
   "repository": "@commonapp/ng-select",
   "engines": {
-    "node": ">= 16.20.0",
-    "npm": ">= 8.19.4"
+    "node": ">= 18.12.0",
+    "npm": ">= 8.19.2"
   },
   "scripts": {
     "build": "ng build ng-select && ng build ng-option-highlight",
     "build:demo": "ng build demo --configuration production --base-href=/ng-select",
+    "build:themes": "node-sass --output-style compressed src/ng-select/themes/ -o dist/ng-select/themes",
+    "copy-sass": "mkdir -p dist/ng-select/scss && cp src/ng-select/**/*.scss dist/ng-select/scss",
+    "copy-examples": "cp -r src/demo/app/examples dist/demo",

```

Comment:
Leave these out - they don't work on our end and we've removed them to avoid confusion.